docs/time/index.rst

```diff
@@ -1013,9 +1013,10 @@ To get the representation of a |Time| object::
   >>> import matplotlib.pyplot as plt  # doctest: +SKIP
   >>> jyear = np.linspace(2000, 2001, 20)  # doctest: +SKIP
   >>> t = Time(jyear, format='jyear')  # doctest: +SKIP
-  >>> plt.plot_date(t.plot_date, jyear)  # doctest: +SKIP
-  >>> plt.gcf().autofmt_xdate()  # orient date labels at a slant  # doctest: +SKIP
-  >>> plt.draw()  # doctest: +SKIP
+  >>> fig, ax = plt.subplots()  # doctest: +SKIP
+  >>> ax.scatter(t.plot_date, jyear)  # doctest: +SKIP
```

Comment:
This misses a `ax.xaxis.axis_date()`, I think.

Though I think that also here, we should perhaps just go for using `t.datetime` to automatically gets dates...